### PR TITLE
Update menu.css

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

**Comment by PalmerAL** [comment]
I like the new animation, but why add the margin between fabs? As far as I can tell, google inbox doesn't have this spacing.

File: editor/css/menu.css [hunk 1]
```diff
@@ -16,27 +16,40 @@
...
+@media (max-height: 520px) {
+	.start-menu {
+		padding-bottom: 0.25em;
+	}
+	.start-menu > a{
+		margin: 0.25em 0.3em;
+	}
+}
+#start-menu-container .start-menu > a {
+	pointer-events: none;
```
Comment (PalmerAL) [inline editor/css/menu.css]: Just curious, why is this needed?

Comment (nt1m) [inline editor/css/menu.css]: This is in case the small fabs are clickable even though they're invisible.

**Comment by nt1m** [comment]
@PalmerAL About the spacing, you're wrong, there's 16px between each fab.

**Comment by PalmerAL** [comment]
Thanks!
